_data-prepper/log-analytics.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: default
+title: Log analytics
+nav_order: 2
+---
+
+# Introduction
+
+Data Prepper is an extendable, configurable, and scalable solution for log ingestion into OpenSearch and Amazon OpenSearch Service. Currently, Data Prepper is focused on receiving logs from [FluentBit](https://fluentbit.io/) via the [Http Source](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/http-source/README.md), and processing those logs with a [Grok Processor](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/grok-processor/README.md) before ingesting them into OpenSearch through the [OpenSearch sink](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/opensearch/README.md).
```

Comment:
Based on #1542, the links for the Data Prepper processors should eventually point to the plugin reference pages. These are not in place yet. So there is no immediate action. But, I want to point out.

Reply:
@dlvenable We will address all of these broken links in their own PR after migration is complete and will comment here with a link to that PR.